graph/src/data/query/error.rs

```diff
@@ -14,15 +19,25 @@ pub enum QueryExecutionError {
     NoRootQueryObjectType,
     NoRootSubscriptionObjectType,
     ResolveEntityError(Pos, String),
+    ResolveListError,
     NonNullError(Pos, String),
     ListValueError(Pos, String),
     NamedTypeError(String),
+    ObjectFieldError,
     AbstractTypeError(String),
     InvalidArgumentError(Pos, String, q::Value),
     MissingArgumentError(Pos, String),
+    StoreQueryError(String),
+    FilterNotSupportedError(String, String),
     UnknownField(Pos, String, String),
     EmptyQuery,
     MultipleSubscriptionFields,
+    SubgraphParseError(String),
+    BuildRangeTypeError,
```

Comment:
Let's do better here. Let's make this `RangeArgumentsError(Vec<String>)` and add the names of all range parameters that are not integers.